### ISSUE TITLE
Reading a PE file on Mac can't find .text section.

### ISSUE THREAD

**Issue by farzonl** [lief-project/LIEF#554]
**Describe the bug**
calling `LIEF::PE::Section &textSection = peBinary->get_section(".text");` results in  [throw LIEF::not_found("No such section with this name");]( https://github.com/lief-project/LIEF/blame/master/src/PE/Binary.cpp#L466)
but if  I print the sections I see the .text section.

```c++
for( auto section : peBinary->sections()) {
      std::cout << section.name() << std::endl;
}
```
output:
```bash
.text
.data
.rdata
.pdata
.xdata
.bss
.idata
.CRT
.tls
/4
/19
/31
/45
/57
/70
/81
/92
```

**To Reproduce**

compile the following assembly 
```bash
nasm -f win32 hello.win.asm
x86_64-w64-mingw32-gcc hello.obj-o hello.win.exe
```
```asm
        global  main
        extern  puts
        section .text
main:
sub     rsp, 28h     
mov     rcx, message 
call    puts         
add     rsp, 28h     
dow space
        ret
message:
db      'Hello', 0   
```

**Expected behavior**
To find the .text section, especially since we can iterate over it.

**Environment (please complete the following information):**
 - System and Version :  ~~Ubuntu 20.04~~ MacOS Mojave 10.14.6 (18G4032)
 - Target format: PE
 - LIEF commit version: 
 ~~https://github.com/lief-project/LIEF/releases/download/0.11.1/LIEF-0.11.1-Linux-x86_64.tar.gz~~
      https://github.com/lief-project/LIEF/releases/download/0.11.3/LIEF-0.11.3-Darwin-x86_64.tar.gz
MD5: ~~b282cc87982a863996c6ac45bd7a02db~~ d033f69581f61386cca64ff0fb9b4e88
git commit: ~980ed6b~ e3ea0d5

**Additional context**
The `get_section` function hasn't change in 4 years, I suspect an underlying data structure is the issue.

**Comment by farzonl** [comment]
You all move quick on releases I will check out 0.11.2 or newer and see if that fixes my issue. https://github.com/lief-project/LIEF/issues/551

**Comment by farzonl** [comment]
fixed in newer releases, closing as dupe of #551

**Comment by farzonl** [comment]
I'm see this issue on Macos now.
